apps/wallet/src/features/Send/TransactionResult/index.tsx

```diff
@@ -22,6 +23,23 @@ const TransactionResult: FC<Props> = ({ navigator }) => {
 		navigator.slideTo(0);
 	};
 
+	const handleBackToHome = () => {
+		navigationRef.reset({
+			index: 1,
+			routes: [
+				{ name: 'Dashboard' },
+				{
+					name: 'Dashboard',
+					params: {
+						screen: 'Home',
+					},
+				},
```

Comment:
I don't know what we call `home`, but if I send an NFT from Solana widget, I prefer to back to the Solana screen instead of to the dashboard. You can get this by querying the latest path, which is also the latest widget

Reply:
idk why this marked as resolved because there is no change from last preview, and I suggest trying `resetRoute` method in `utils/navigation/index.ts`

Reply:
This function is used for the Button "Back to home", so its mission is to nav back to the `Home` screen.

`resetRoute` is a better approach, I have tweaked the code to use that method